components/builder-worker/src/runner/mod.rs

```diff
@@ -69,6 +69,12 @@ lazy_static! {
     static ref STUDIO_PKG: PackageIdent = PackageIdent::from_str("core/hab-studio").unwrap();
 }
 
+macro_rules! bldr_channel_name {
```

Comment:
Should this go into the core crate instead of being duplicated?

Also, does it need to be a macro?